src/main/java/org/cryptomator/ui/unlock/UnlockInvalidMountPointController.java

```diff
@@ -25,26 +30,32 @@ public class UnlockInvalidMountPointController implements FxController {
 	private final FxApplicationWindows appWindows;
 	private final ResourceBundle resourceBundle;
 	private final ExceptionType exceptionType;
+	private final Path exceptionPath;
 	private final String exceptionMessage;
+	private final Path hideawayPath;
 
 	public FormattedLabel dialogDescription;
 
 	@Inject
-	UnlockInvalidMountPointController(@UnlockWindow Stage window, @UnlockWindow Vault vault, @UnlockWindow AtomicReference<Throwable> unlockException, FxApplicationWindows appWindows, ResourceBundle resourceBundle) {
+	UnlockInvalidMountPointController(@UnlockWindow Stage window, @UnlockWindow Vault vault, @UnlockWindow AtomicReference<IllegalMountPointException> illegalMountPointException, FxApplicationWindows appWindows, ResourceBundle resourceBundle) {
 		this.window = window;
 		this.vault = vault;
 		this.appWindows = appWindows;
 		this.resourceBundle = resourceBundle;
 
-		var exc = unlockException.get();
+		var exc = illegalMountPointException.get();
 		this.exceptionType = getExceptionType(exc);
+		this.exceptionPath = exc.getMountpoint();
 		this.exceptionMessage = exc.getMessage();
+		this.hideawayPath = exc instanceof HideawayAlreadyExistsException haeExc ? haeExc.getHideaway() : null;
```

Comment:
isn't the constructor invoked, before the exception is set?

We usually use Properties that can react to changes.

Reply:
> isn't the constructor invoked, before the exception is set?

The constructor is only invoked here:
https://github.com/cryptomator/cryptomator/blob/614756c7408f511bea4d92ac265538c85c68b1ae/src/main/java/org/cryptomator/ui/unlock/UnlockWorkflow.java#L82-L83
`invalidMountpointScene` is lazy. 

> We usually use Properties that can react to changes.

In this case `illegalMountPointException` hasn't been used like that historically. According to @infeo the usage of AtomicReference is caused by a need for passing a reference.
Changing this setup would introduce a lot of boiler plate IMAO.

Reply:
Arguing with history is never a good idea 😅 

I actually see the point of @overheadhunter, it would make the usage more safe. And the boilerplate code would only contain a `Platform.runLater(() -> exception.set(throwable)), or am I wrong? Because a property is bascially an observable container.

Reply:
What I meant: Currently there is no demand for reacting to change. I can implement it, but there is no demand for it. :D

I would have to move away from `final fields` to a property-based approach. I will look into it.

Reply:
Resolved by 0b6782d44bc8c8efdae20fb9626e770fa9c65bd1.